src/main/java/com/google/sps/data/User.java

```diff
@@ -1,11 +1,15 @@
 package com.google.sps.data;
 
+import java.util.ArrayList;
+
 public final class User {
 
   private final String name;
+  private final ArrayList<String> productArea;
```

Comment:
Maybe use the Collection interface here instead? This will allow you to more easily change the implementation.

Reply:
Good point, done